### PR TITLE
INS-19097-Change-doc-for-ODES/ES

### PR DESCRIPTION
**Pull request by yiwei-wu-instaclustr**
Update the terraform doc to retire support for ODES/ES

### CODE REVIEW

**Comment by iclonghoang** [comment]
there's also table `Bundles and Versions` in `Readme.md` which we could add 'For Legacy Support Only' notation

**Comment by yiwei-wu-instaclustr** [comment]
> there's also table `Bundles and Versions` in `Readme.md` which we could add 'For Legacy Support Only' notation

> Updated, thanks Long

File: acc_test/data/valid_with_resizable_opensearch_cluster.tf [hunk 1]
```diff
@@ -20,7 +20,7 @@ resource "instaclustr_cluster" "resizable_cluster" {
   }
   bundle {
     bundle  = "OPENSEARCH"
-    version = "1.2.4"
+    version = "1.3.1"
```
Comment (iclonghoang) [inline acc_test/data/valid_with_resizable_opensearch_cluster.tf:23]: with this version change, can you please enable this 1.3.1 for [tf test account ](https://instaclustr.atlassian.net/wiki/spaces/IDS/pages/1623425094/Terraform+Provider+Test+Account)

this is preemptive action for subsequent opensearch version bump where 1.3.1 becomes disabled, which will fail this test

Comment (yiwei-wu-instaclustr) [reply acc_test/data/valid_with_resizable_opensearch_cluster.tf:23]: That's a very good reminder. Thanks Long

**Review by iclonghoang — APPROVED** [review]

File: acc_test/resource_cluster_test.go [hunk 2]
```diff
@@ -320,8 +320,8 @@ func TestAccOpenSearchClusterResize(t *testing.T) {
 	hostname := getOptionalEnv("IC_API_URL", instaclustr.DefaultApiHostname)
 	resourceName := "resizable_cluster"
 	oriConfig := fmt.Sprintf(string(validConfig), username, apiKey, hostname)
-	validResizeConfig := strings.Replace(oriConfig, `opensearch_dashboards_node_size = "t3.small-v2",`, `opensearch_dashboards_node_size = "m5xl-400-v2",`, 1)
-	invalidResizeConfig := strings.Replace(oriConfig, `opensearch_dashboards_node_size = "t3.small-v2",`, `opensearch_dashboards_node_size = "t3.small",`, 1)
+	validResizeConfig := strings.Replace(oriConfig, `opensearch_dashboards_node_size = "SRH-DEV-t4g.small-5"`, `opensearch_dashboards_node_size = "SRH-DEV-t4g.small-30"`, 1)
+	invalidResizeConfig := strings.Replace(oriConfig, `opensearch_dashboards_node_size = "SRH-DEV-t4g.small-5"`, `opensearch_dashboards_node_size = "SRH-DEV-t4g.small"`, 1)
```
Comment (niluka-insta) [inline acc_test/resource_cluster_test.go:324]: typo in `SRH-DEV-t4g.small` ?

File: acc_test/resource_cluster_test.go [hunk 3]
```diff
@@ -342,7 +342,7 @@ func TestAccOpenSearchClusterResize(t *testing.T) {
 					time.Sleep(1 * time.Minute)
 				},
 				Config:      invalidResizeConfig,
-				ExpectError: regexp.MustCompile("t3.small"),
+				ExpectError: regexp.MustCompile("SRH-DEV-t4g.small"),
```
Comment (niluka-insta) [inline acc_test/resource_cluster_test.go:345]: typo in `SRH-DEV-t4g.small` ?

Comment (yiwei-wu-instaclustr) [reply acc_test/resource_cluster_test.go:345]: What typo? this is to match the invalid test case for opensearch cluster

File: acc_test/resource_cluster_test.go (hunk 2 above)
Comment (yiwei-wu-instaclustr) [reply acc_test/resource_cluster_test.go:324]: no? just an invalid testcase

File: docs/resources/firewall_rule.md [hunk 4]
```diff
@@ -19,9 +19,9 @@ Property | Description | Default
 
 #### rules
 
-Property | Description | Default
----------|-------------|--------
-`type`|Accepts CASSANDRA, SPARK, SPARK_JOBSERVER, APACHE_ZOOKEEPER, KAFKA, KAFKA_CONNECT, ELASTICSEARCH, REDIS|Required
+Property | Description                                                                                                                                        | Default
+---------|----------------------------------------------------------------------------------------------------------------------------------------------------|--------
+`type`| Accepts CASSANDRA, SPARK, SPARK_JOBSERVER, APACHE_ZOOKEEPER, KAFKA, KAFKA_CONNECT, ELASTICSEARCH, REDIS (ELASTICSEARCH is for legacy support only) |Required
```
Comment (niluka-insta) [inline docs/resources/firewall_rule.md:24]: Doesn't really matter, would have preferred if it was like,

`Accepts CASSANDRA, SPARK, SPARK_JOBSERVER, APACHE_ZOOKEEPER, KAFKA, KAFKA_CONNECT, ELASTICSEARCH(Only for legacy support), REDIS`

**Review by niluka-insta — APPROVED** [review]
LGTM

**Pull request merged**